auto-value-gson/src/test/java/com/ryanharter/auto/value/gson/AutoValueGsonExtensionTest.java

```diff
@@ -143,62 +142,33 @@ public void simple() {
         + "@Generated(\n"
         + "    value = \"com.ryanharter.auto.value.gson.AutoValueGsonExtension\",\n"
         + "    comments = \"https://github.com/rharter/auto-value-gson\"\n"
-        + ")"
+        + ")\n"
         + "final class AutoValue_Test extends $AutoValue_Test {\n"
-        + "  AutoValue_Test(String a, int[] b, int c, String d, @Nullable String e, ImmutableMap<String, "
-        + "Number> f,\n"
-        + "      Set<String> g, Map<String, Set<String>> h, String i, @Nullable List<String> j, String k, "
-        + "List<String> l,\n"
-        + "      String m, List<String> n,\n"
+        + "  AutoValue_Test(String a, int[] b, int c, String d, @Nullable String e,\n"
+        + "      ImmutableMap<String, Number> f, Set<String> g, Map<String, Set<String>> h, String i,\n"
+        + "      @Nullable List<String> j, String k, List<String> l, String m, List<String> n,\n"
         + "      Map<String, Map<String, Map<String, Map<String, Map<String, String>>>>> o) {\n"
         + "    super(a, b, c, d, e, f, g, h, i, j, k, l, m, n, o);\n"
         + "  }\n"
         + "\n"
         + "  public static final class GsonTypeAdapter extends TypeAdapter<Test> {\n"
-        + "    private final TypeAdapter<String> aAdapter;\n"
-        + "    private final TypeAdapter<int[]> bAdapter;\n"
-        + "    private final TypeAdapter<Integer> cAdapter;\n"
-        + "    private final TypeAdapter<String> dAdapter;\n"
-        + "    private final TypeAdapter<String> eAdapter;\n"
-        + "    private final TypeAdapter<ImmutableMap<String, Number>> fAdapter;\n"
-        + "    private final TypeAdapter<Set<String>> gAdapter;\n"
-        + "    private final TypeAdapter<Map<String, Set<String>>> hAdapter;\n"
-        + "    private final TypeAdapter<String> iAdapter;\n"
-        + "    private final TypeAdapter<List<String>> jAdapter;\n"
-        + "    private final TypeAdapter<String> kAdapter;\n"
-        + "    private final TypeAdapter<List<String>> lAdapter;\n"
-        + "    private final TypeAdapter<String> mAdapter;\n"
-        + "    private final TypeAdapter<List<String>> nAdapter;\n"
-        + "    private final TypeAdapter<Map<String, Map<String, Map<String, Map<String, "
-        + "Map<String, String>>>>>> oAdapter;\n"
+        + "    private final TypeAdapter<String> java_lang_StringAdapter;\n"
+        + "    private final TypeAdapter<int[]> int__Adapter;\n"
```

Comment:
Some of these names don't quite come out perfectly readible, but I think it's sort of ok since it's generated code and the TypeNames themselves still hash correctly